nvim/plugin/packer_compiled.lua

```diff
@@ -0,0 +1,162 @@
+-- Automatically generated packer.nvim plugin loader code
```

Comment:
if it's auto generated maybe it's safe to ignore

Reply:
Yeah it was an oversight will ignore